Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -11,6 +11,9 @@
 
 #include "StdInc.h"
 #include <net/SyncStructures.h>
+#include <../Client/game_sa/CAnimBlendAssocGroupSA.h>
+#include <../Client/game_sa/CAnimBlendAssociationSA.h>
```

Comment:
GameSA headers should not be used here. Instead, add an interface to the Game SDK (/Client/sdk/game).

Reply:
Good catch. I forgot to ask you what to do with them. I'll fix them.